07_deploy_masters.sh

```diff
@@ -37,6 +37,7 @@ for node in $(jq -r .nodes[].name $MASTER_NODES_FILE); do
   # FIXME(shardy) we should parameterize the image
   openstack baremetal node set $node --instance-info "image_source=http://172.22.0.1/images/$RHCOS_IMAGE_FILENAME_LATEST" --instance-info image_checksum=$(curl http://172.22.0.1/images/$RHCOS_IMAGE_FILENAME_LATEST.md5sum) --instance-info root_gb=25 --property root_device="{\"name\": \"$ROOT_DISK\"}"
   openstack baremetal node manage $node --wait
+  openstack baremetal node inspect $node --wait
```

Comment:
In my local env this adds over 8 minutes to the runtime for this script, could you do them in parallel similar to how we do the deploy so it should only add 3 minutes or so?

Reply:
Yeah, waiting for each node is a waste of time. We probably need to start inspection for each, then loop waiting for "manageable" again.

Reply:
Fixed to add loop to check for manageable.